docs/CHANGELOG.md

```diff
@@ -38,6 +38,14 @@ Order = 5
 - Add new config parameter to OCR and OCR2 named `TraceLogging` that enables trace logging of OCR and OCR2 jobs, previously this behavior was controlled from the `P2P.TraceLogging` parameter. To maintain the same behavior set `OCR.TraceLogging` and `OCR2.TraceLogging` to the same value `P2P.TraceLogging` was set.
 
 - Add two new config parameters `WebServer.ListenIP` and `WebServer.TLS.ListenIP` which allows binding Chainlink HTTP/HTTPS servers to a particular IP. The default is '0.0.0.0' which listens to all IP addresses (same behavior as before). Set to '127.0.0.1' to only allow connections from the local machine (this can be handy for local development).
+- Add several new metrics for mercury feeds, related to WSRPC connections:
+    - `mercury_transmit_timeout_count`
+    - `mercury_dial_count`
+    - `mercury_dial_success_count`
+    - `mercury_dial_error_count`
+    - `mercury_connection_reset_count`
```

Comment:
Are we committed to the `mercury_` prefix, rather than e.g. `llo_`?

Reply:
This is already used for other metrics e.g. `mercury_transmit_success_count`, kept as `mercury` to be consistent.

I don't think there is any pressing need to change the name we use internally.